Sources/SwiftFusion/Geometry/Pose2.swift

```diff
@@ -133,10 +133,12 @@ extension Pose2Coordinate: LieGroupCoordinate {
   public func inverse() -> Pose2Coordinate {
     Pose2Coordinate(self.rot.inverse(), self.rot.unrotate(-self.t))
   }
+}
 
+extension Pose2Coordinate {
   /// Group action on `Vector2`.
   @differentiable
-  public static func * (lhs: Pose2Coordinate, rhs: Vector2) -> Vector2 {
+  static func * (lhs: Pose2Coordinate, rhs: Vector2) -> Vector2 {
```

Comment:
Did you remove `public` intentionally? I think this should be public.

Reply:
Oh actually I see that it is the "Coordinate". Non-public seems good.

Reply:
Yeah, I had been thinking whether or not to declare it public, since from what I gathered the "...Coordinate" is actually an implementation detail of the actual type.